tools/topology/topology1/m4/dts_codec_adapter.m4

```diff
@@ -10,8 +10,6 @@ define(`CA_SETUP_CONTROLBYTES',
 define(`CA_SETUP_CONTROLBYTES_MAX', 8192)
 define(`CA_SETUP_CONTROLBYTES_NAME', `DTS Codec Setup ')
 
-define(`CA_SCHEDULE_CORE', 0)
-
```

Comment:
spurious change?

Reply:
I think so. DTS module doesn't have the particular demand, CA_SCHEDULE_CORE should be assigned as SCHEDULE_CORE in the latter code.